npm/webpack-preprocessor/README.md

```diff
@@ -42,12 +42,6 @@ module.exports = (on) => {
 }
 ```
 
-## Examples
```

Comment:
Would it be worth listing the system-test examples we have in place?

Reply:
I've added a few examples back.